infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/DraftableObjectDao.kt

```diff
@@ -57,10 +61,45 @@ interface IDraftableObjectDao<T : Draftable<T>> : IDraftableObjectReader<T>, IDr
 
 @Transactional(readOnly = true)
 abstract class DraftableDaoBase<T : Draftable<T>>(
-    jdbcTemplateParam: NamedParameterJdbcTemplate?,
+jdbcTemplateParam: NamedParameterJdbcTemplate?,
     private val table: DbTable,
 ): DaoBase(jdbcTemplateParam), IDraftableObjectDao<T> {
 
+    @Transactional
+    override fun updateAndVerifyVersion(previousVersion: RowVersion<T>, updatedItem: T): RowVersion<T> =
```

Comment:
Tämä jäi ajatuksena vähän puoliväliin ja on nyt käytössä vain julkaisussa. Mutta ajattelin että jos tämän remppaisi siihen kuosiin että insert & update checkaisi versiot, tällä voisi logittaa mahdolliset katoavat päällekäis-muutokset. Toki hyöty vähän ehkä pienehkö sillä saa kiinni vain jos se on päällekäinen backarin kutsu - ei jos se tila oli killumassa UI:n puolella. Sitä varten pitäisi kierrättää versiot frontin kautta dialogeilla.